const throwError = (error, errorMessage) => {
  return new Error(errorMessage);
};

export default throwError;
